static/app/views/automations/components/automationListRow.tsx

```diff
@@ -49,12 +35,7 @@ export function AutomationListRow({
           }}
         />
         <CellWrapper>
-          <StyledTitleCell
-            name={name}
-            link={link}
-            details={details}
-            disabled={disabled}
-          />
+          <TitleCell href={`${AUTOMATIONS_BASE_URL}/${id}/`}>{name}</TitleCell>
```

Comment:
links within the app should use our Link component

Reply:
fixed in https://github.com/getsentry/sentry/pull/90422/commits/4c775b762cc3ca2cb53f8765ff90ecc22d4fa4eb